net/interfaces/interfaces_darwin_nocgo.go

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !cgo
```

Comment:
I think this needs to be `+build darwin,!nocgo` ... the filename doesn't match per:

https://golang.org/pkg/go/build/

> If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns: ...